<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubmoduleAddDialog</class>
 <widget class="QDialog" name="SubmoduleAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clone</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Remote</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_test">
       <property name="text">
        <string>&amp;Test</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="DirectoryLineEdit" name="lineEdit_working_dir"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Local</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="RepositoryLineEdit" name="lineEdit_repo_location">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_force">
        <property name="text">
         <string>--force</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="RemoteAdvancedOptionWidget" name="advanced_option" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RepositoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>RepositoryLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>DirectoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>DirectoryLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>RemoteAdvancedOptionWidget</class>
   <extends>QWidget</extends>
   <header>RemoteAdvancedOptionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_repo_location</tabstop>
  <tabstop>pushButton_test</tabstop>
  <tabstop>lineEdit_working_dir</tabstop>
  <tabstop>pushButton_browse</tabstop>
  <tabstop>checkBox_force</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SubmoduleAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SubmoduleAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
